api-docs/cloud-dedicated/ref.yml

```diff
@@ -1863,13 +1917,30 @@ paths:
         required: true
       responses:
         "204":
-          description: Write data is correctly formatted and accepted for writing to the database.
+          description: |
+            Success. All points in the batch are written and queryable.
         "400":
```

Comment:
201 when partial-writes are on, and some lines were written.
Otherwise, still a 400.